core/scc/cscc/configure.go

```diff
@@ -244,7 +246,7 @@ func validateConfigBlock(block *common.Block) error {
 // Since it is the first block, it is the genesis block containing configuration
 // for this chain, so we want to update the Chain object with this info
 func joinChain(chainID string, block *common.Block, sccp sysccprovider.SystemChaincodeProvider, deployedCCInfoProvider ledger.DeployedChaincodeInfoProvider, lr plugindispatcher.LifecycleResources, nr plugindispatcher.CollectionAndLifecycleResources) pb.Response {
-	if err := peer.CreateChainFromBlock(block, sccp, deployedCCInfoProvider, lr, nr); err != nil {
+	if err := xpeer.HandleCreateChainFromBlock(chainID, peer.CreateChainFromBlock)(block, sccp, deployedCCInfoProvider, lr, nr); err != nil {
```

Comment:
is this correct?

according to .HandleCreateChainFromBlock(c...) it returns the second argument unchanged. (line 39 of extensions/peer/peer.go)

Reply:
Yes, in case of mod it should work as it works now, in peer-ext we are going to provide actual implementation for this function